packages/@ourworldindata/grapher/src/tooltip/Tooltip.scss

```diff
@@ -1,467 +1,496 @@
                     }
-                    color: $grey;
-                }
-            }
-        }
 
-        .variable + .variable {
-            margin-top: 4px;
-            padding-top: 8px;
-            border-top: 1px solid $light-grey;
-        }
+                    // -- special row types --
 
-        table.series-list {
-            color: $dark-grey;
-            font-size: 14px;
-            line-height: 22px;
-            font-weight: $medium;
-            white-space: normal;
-            border-collapse: collapse;
-            width: 100%;
-
-            // only used if rows have ≥2 values
-            thead {
-                font-size: 12px;
-                letter-spacing: 0.01em;
-
-                tr td.series-value {
-                    font-weight: $medium;
-                }
-            }
+                    tr.blurred {
+                        color: $light-grey;
+                        .series-color .swatch {
+                            opacity: 0.25;
+                        }
+                        .series-name span {
+                            color: inherit;
+                        }
+                    }
 
-            // -- standard columns --
-            td {
-                vertical-align: baseline;
-            }
+                    tr.spacer {
+                        line-height: 2px;
+                        font-size: 2px;
+                        &::before {
+                            content: "\00a0";
+                        }
+                    }
 
-            td.series-color {
-                padding-left: 0;
-                .swatch {
-                    width: 12px;
-                    height: 12px;
-                    display: inline-block;
-                    margin-right: 0.3em;
-                    text-align: left;
-                    position: relative;
-                }
-            }
+                    tr.total {
+                        td {
+                            line-height: 14px;
+                        }
+                        td:nth-child(2),
+                        td:nth-child(3) {
+                            border-top: 1px solid $background-stroke;
+                            vertical-align: bottom;
+                        }
+                        td:last-child::before {
+                            content: "\200a";
+                            height: 5px;
+                            display: block;
+                        }
+                    }
 
-            td.series-name {
-                padding-right: 0.9em;
-                line-height: 16px;
-                width: 100%;
-                .parenthetical {
-                    color: $grey;
-                }
+                    tr.total--top {
+                        td:nth-child(2),
+                        td:nth-child(3) {
+                            border-bottom: 1px solid $background-stroke;
+                            vertical-align: top;
```

Comment:
In what must be absolute outrage for Marwa, the "Total" line is not correctly aligned.

See this screenshot, where the ruler is touching the "T" of "Total", but not the number behind it.

Don't know why this is - but getting rid of this `vertical-align` line gets rid of it.

![CleanShot 2024-11-07 at 16 28 04](https://github.com/user-attachments/assets/e7175be5-b12e-46d9-98ad-907ee17a5474)